src/im/impage.cpp

```
@@ -1584,15 +1504,16 @@ bool ImPage::FindText( )
+    int region_count = 0;
+    imAnalyzeFindRegions(m_opImMain, m_opImTmp1, 4, 1, &region_count);
     if (!region_count)
         return this->Terminate( ERR_NONE );
 
     // calcul des centroids
-    float* cx = (float*)malloc(region_count*sizeof(float));
-    memset(cx, 0, region_count*sizeof(float));
-    float* cy = (float*)malloc(region_count*sizeof(float));
-    memset(cy, 0, region_count*sizeof(float));
+    double* cx = (double*)malloc(region_count*sizeof(double));
+    memset(cx, 0, region_count*sizeof(double));
```

Cppcheck (2.21.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)